client/post-editor/editor-seo/index.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import CountedTextarea from 'components/forms/counted-textarea';
+import Gridicon from 'components/gridicon';
+import InfoPopover from 'components/info-popover';
+import TokenField from 'components/token-field';
+
+function getSeoTabUrl( slug ) {
+	return `/settings/seo/${ slug }`;
+}
+
+export default React.createClass( {
+	displayName: 'EditorSeo',
+
+	propTypes: {
+		site: React.PropTypes.object.isRequired,
+		post: React.PropTypes.object.isRequired
+	},
+
+	render() {
+		const { slug } = this.props.site;
```

Comment:
We should try to connect components as much as possible using Redux state now, avoiding the need to pass around objects like `site` and reducing our dependency on the legacy `lib/sites-list` module. Specifically for this component, you can `connect` using the [`getSiteSlug` selector](https://github.com/Automattic/wp-calypso/blob/7afa9ae54c31bfe7bc528fa6f607bfe8079f35ca/client/state/sites/selectors.js#L78-L96). See ["App Components" section of "Our Approach to Data"](https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#app-components).